sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -81,13 +90,14 @@ def online_read(
         table: FeatureView,
         entity_keys: List[EntityKeyProto],
         requested_features: List[str] = None,
-    ) -> List[Tuple[Optional[datetime], Optional[Dict[str, ValueProto]]]]:
+    ):
```

Comment:
can we maintain the return type? you can probably just return `[]` if there is no online store